src/components/questions/Answer.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import timeAgo from "../../utils/timeAgo.js";
+
+
+const Answer = ({ answer }) => {
+  return (
+    <li>
+      <span>{ answer.body }</span>
+      <div className="help-container">
+        <span>
+          {
+            (answer.answerer_name === 'Seller')?
+              (<strong>by: { answer.answerer_name }</strong>)
+              :(<>by: { answer.answerer_name }</>)
+          }
+        </span>
+        <span>{ timeAgo(answer.date) }</span>
+        <span> | </span>
+        <span> Helpful ? <a> Yes { answer.helpfulness } </a> </span>
+        <span> | </span>
+        <a> Report </a>
+      </div>
+      <div className="photo-container">
+        {
+          (answer.photos)?
+          (answer.photos.map((photo) => {
+              return <img key={ photo.id } src={ photo.url } />;
+          }))
+          :null
```

Comment:
In my opinion, I think for the conditional rendering, when the condition is false and there's nothing you want to render for the false condition like null, you can use as the below. 

You don't have to change but might consider next time. I personally use as the below when I want to render only if the condition is true.

{(condition) && (result, only if the condition is true)}

Example :
{(answer.photos) && (
      (answer.photos.map((photo) => {
          return <img key={ photo.id } src={ photo.url } />;
      }))
}

Reply:
Like this a lot!